pages/documentation/developer/repositories/powsybl-single-line-diagram.md

```diff
@@ -6,11 +6,20 @@ This [repository](https://github.com/powsybl/powsybl-single-line-diagram) provid
 
 ## Features
 
-- creation of single line diagrams for given voltage levels
+- creation of single line diagrams for given voltage levels, substations or zones in SVG format
+- diagrams customization
+     - several layout algorithms to generate the diagrams
+     - several layout parameters to adjust the rendering
+     - use your own component library or modify the existing
+- small demo app to display the diagrams (note that this app is mainly for debug purposes and therefore is not included in the release)
+- extensions to force positions in the diagram 
+
+// TODO: complete the list
```

Comment:
Do not use this syntax, use the `<span style="color: red">TODO: complete the list</span>` syntax like everywhere.
`//` doesn't comment anything if it's what you're trying to do